scripts/sentiment_analysis/process.py

```diff
@@ -0,0 +1,126 @@
+"""Load various datasets."""
+
+import re
+import multiprocessing as mp
+import time
+from functools import partial
+
+import gluonnlp as nlp
+from mxnet import nd, gluon
```

Comment:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

You should put a blank line between each group of imports. Sort imports alphabetically within each group.